docs/manual/manual_advanced_arming.rst

```diff
@@ -0,0 +1,756 @@
+
+The *Disarm Throttle Upper Limit* defines the upper, more positive bound of the disarming throttle region. It should always be more positive than or equal to the 
+*Disarm Throttle Lower Limit*. The *Disarm Throttle Lower Limit* defines the lower bound of the disarming throttle region. **Any throttle command from the same** :ref:`source <throttle_sources>`
+**as the module armed with that the module receives in this region is considered a disarming throttle and will cause a disarming transition.** For more information on 
+the throttle source requirements for disarming, see :ref:`disarming_throttle_source_requirement` below. Note that unlike with arming throttles, it always only takes one 
+disarming throttle to trigger a disarm. This is to allow disarms to happen as quickly as possible in critical safety situations.
+
+The image below illustrates where the disarming throttle region would be for an example setup where the *Disarm Throttle Upper Limit* is set to 0.15 and the 
+*Disarm Throttle Lower Limit* is set to 0.0. Any throttle command between 0% and 15% is a disarming throttle in this setup.
+
+.. figure:: ../_static/manual_images/arming/disarming_throttle_region_example_1.png
+    :align: center
+    :width: 60%
+    :alt: Example Disarming Throttle Region
+
+    Example Disarming Throttle Region
+
+**There is a separate parameter that determines whether the disarming throttle region exists at all. The *Disarm On Throttle* parameter, shown below in IQ Control Center, 
+allows users to toggle the disarming throttle region on or off completely.** If this parameter is set to *Do Not Disarm on Throttle*, then there is no disarming throttle region 
+and throttles cannot cause an disarming transition. Otherwise, disarming throttles can be used as previously described.
+
+.. figure:: ../_static/manual_images/arming/disarm_on_throttle_parameter.png
+    :align: center
+    :width: 60%
+    :alt: Disarm on Throttle Parameter
+
+    Disarm on Throttle Parameter in IQ Control Center
+
+.. _disarming_throttle_source_requirement:
+
+Disarming Throttle Source Requirement
+--------------------------------------
+**To disarm with a throttle command that throttle command must be from the same** :ref:`source <throttle_sources>` **as the throttle commands that armed the module.** This is to reduce the 
+chance of noise on communication interfaces that are not being actively used from unintentionally triggering a disarm.
+
+For example, if a module armed using throttle commands sent over DroneCAN, then any throttle commands sent using a different communication protocol, such as :ref:`hobby_standard_pwm`, 
+could not trigger a disarm. Throttle commands from a different source will have no effect on the armed state of the module and will not change how the module is spinning.
+
+Disarm Example
+---------------
+An example of a possible disarming setup and procedure is outlined below to clarify how disarming with throttles works.

```

Comment:
don't need the to clarify how disarming with throttles works...that's what an example is